### DISCUSSION TITLE
Mispelling on the 'list of project'

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#3793] [Bugs]
I think it doesn't need any issue for that but there is a mispelling on the pop up (cf image) on the 'QfieldCloud project list'

Should be 'statut du cloud' and not 'statut du coloud'

![2023-01-05 11_25_56-](https://user-images.githubusercontent.com/7015197/210758439-091d068e-30c0-4be0-b771-b22f43394b59.png)

**Comment by suricactus** [comment]
Hey, @GlaDal . Thanks for the sharp eye! While OPENGIS.ch has multilingual team, most of the translations are done by the community via Transifex. Feel free to join the translators community here https://explore.transifex.com/opengisch/qfieldsync/.
